lib/tsconfig/4.8.x/ng14/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "../tsconfig.json",
+  "angularCompilerOptions": {
+    "allowEmptyCodegenFiles": false,
+    "annotateForClosureCompiler": false,
+    "annotationsAs": "static fields",
+    "disableExpressionLowering": true,
+    "disableTypeScriptVersionCheck": false,
+    "enableIvy": true,
+    "enableLegacyTemplate": false,
+    "enableResourceInlining": false,
+    "fullTemplateTypeCheck": true,
+    "generateCodeForLibraries": true,
+    "preserveWhitespaces": false,
+    "skipMetadataEmit": false,
+    "strictMetadataEmit": false,
+    "skipTemplateCodegen": false,
+    "strictTemplates": true,
+    "strictInjectionParameters": true
```

Comment:
By default, angular add the rule `"strictInputAccessModifiers": true`. 
Could you test it and adapt the configuration ?